CHANGELOG.md

```diff
@@ -4,6 +4,15 @@ The changelog for `ReactiveLists`. Also see the [releases](https://github.com/pl
 
 ------
 
+0.2.0
+-----
+
+- Improve semantics of row height API
```

Comment:
let's link the issue number here and author:

- Improve semantics of row height API ([#44](https://github.com/plangrid/ReactiveLists/issues/41), [@asmallteapot](https://github.com/asmallteapot))

```markdown
- Improve semantics of row height API ([#44](https://github.com/plangrid/ReactiveLists/issues/41), [@asmallteapot](https://github.com/asmallteapot))
```